GraphicEditorTests/BuildingSearchesTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using GraphicEditor.Repositories;
+using GraphicEditor.Repositories.Interfaces;
+using Xunit;
+
+namespace GraphicEditorTests
+{
+    public class BuildingSearchesTests
+    {
+        private readonly IBuildingRepository _buildingRepository;
+
+        public BuildingSearchesTests()
+        {
+            // Arrange
+            _buildingRepository = new BuildingRepository();
+        }
+
+        [Fact]
+        public void GetBuildingsByName_BuildingsExist_ReturnBuildings()
+        {
+            // Act
+            var buildings = _buildingRepository.GetBuildingsByName("Cardiology");
+
+            // Assert
```

Comment:
Although not necessary these comments are fine by me as they can help you when writing tests, I also use them and encourage their use (you can also write given, when, then if you prefer) 😄